apps/api/v2/src/modules/destination-calendars/inputs/destination-calendars.input.ts

```diff
@@ -2,16 +2,25 @@ import { ApiProperty } from "@nestjs/swagger";
 import { Expose } from "class-transformer";
 import { IsString } from "class-validator";
 
+import {
+  APPLE_CALENDAR_TYPE,
+  GOOGLE_CALENDAR_TYPE,
+  OFFICE_365_CALENDAR_TYPE,
+} from "@calcom/platform-constants";
+
 export class DestinationCalendarsInputBodyDto {
   @IsString()
```

Comment:
We still need to use @IsEnum here - you specified type of `integration` that is correct, but its still possible to make a request with integration "random" and it will pass the validation.